diff --git a/dist/bindings/getLlama.js b/dist/bindings/getLlama.js
index e9cb280..2da67ac 100644
--- a/dist/bindings/getLlama.js
+++ b/dist/bindings/getLlama.js
@@ -572,7 +572,8 @@ function getShouldTestBinaryBeforeLoading({ isPrebuiltBinary, platform, platform
             return true;
     }
     else if (platform === "win") {
-        if (buildMetadata.buildOptions.gpu !== false)
+        if (process.env.VAULT_APPCONTAINER_LOCKED !== "1" &&
+            buildMetadata.buildOptions.gpu !== false)
             return true;
     }
     return false;
